packages/pyright-internal/src/backgroundAnalysisBase.ts

```diff
@@ -136,7 +137,7 @@ export class BackgroundAnalysisBase {
         const cancellationId = getCancellationTokenId(token);
         this.enqueueRequest({
             requestType: 'getDiagnosticsForRange',
-            data: { filePath, range, cancellationId },
+            data: { fileUri: fileUri.toString(), range, cancellationId },
```

Comment:
I see, you need this since uri is not pure `object` but has `class implementation`

Reply:
If it was a pure object would this somehow just call 'toString' automatically?